content/posts/2021/01/2021-6-7-tech-communities/index.md

```diff
@@ -0,0 +1,199 @@
+
+### 15. Atlassian Community
+
+This is a community of Atlassian tools lovers and Enthusiasts converging to discuss emerging topics and Technologies revolving around Atlassian Tools like Confluence, Jira, BitBucket among Others. Checkout [<span style="color:blue"> **Official Atlassian Twitter**</span>] (https://twitter.com/Atlassian) and [<span style="color:blue"> **this**</span>] (https://twitter.com/ACNairobi) for various events or meetup updates.
+
+### 16. Codennovation
+
+This is a community of great leaders and mentors aimiming to empower the youth digitally through various Technologies. Check it out [<span style="color:blue"> **here**</span>](https://codenovation.org/).
+
+### 17. AI Kenya
+
+This is a community of Machine Learning, Data Science and Robotics Enthusiasts and Experts. They aim to enable the growth of these technologies in Kenya. Check them out [<span style="color:blue"> **here**</span>](https://twitter.com/AiKenya1).
+
+### 18. Digital Ocean Community
+
+This is an amazing community of Developers empowering each other through various tutorials. Check them out [<span style="color:blue"> **here**</span>] (https://www.digitalocean.com/community)
+
+### 19. Android254
+
+This is a Community of Android Developers who meet to discuss various Android Technologies and Best Practices. Check it out  [<span style="color:blue"> **here**</span>](https://twitter.com/254androiddevs?lang=en)
+
+### 20. Hackernoon
+
+Hackernoon reflects the Technology industry with opinions and stories written by Real Tech Professionals. It covers a variety of Tech topics with interestng conversations. Check it out [<span style="color:blue"> **here**</span>]( https://hackernoon.com/)
+
+### 21. AWS User Group
+
+This contains various AWS(Amazon Web Services)  User Groups from all over the world, discussing Topics around AWS.  Check it out [<span style="color:blue"> **here**</span>](https://aws.amazon.com/developer/community/usergroups/) and see the community in your region that would suit you.
+
+### 22. Product Tank
+
+This is a community of Product Enthusiasts eg Product Owners, Product Managers amongst other specializations who meet to discuss various topics around Product. Check it out [<span style="color:blue"> **here**</span>](https://www.mindtheproduct.com/producttank/) to be part of the Community.
+
+### 23. Product School
+
+Product School offers trainings to Product Enthusiasts and prepares students for product management interviews to enable them expand their Proffesssional Network through access to a Private Community of Product Experts. Find out more
+[<span style="color:blue"> **here**</span>](https://productschool.com/).
+
+### 24. DigiClan Africa
+
+This is a community of Digital Marketing Professionals and Enthusiasts who focus on training people to be better Content creators through Digital Marketing. Check it out [<span style="color:blue"> **here**</span>](https://www.digiclan.africa/).
+
+### 25. Reddit
```

Comment:
Replace this with HackerNews [link](https://news.ycombinator.com)

Reddit is like a Social media app

Reply:
@TheBeachMaster Ain't Hackernews also a social site as well?

Reply:
It's a discussion board for techies... Reddit has a Twitter-like feel